kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -252,7 +252,6 @@ AUTH_METHOD="client_secret"
 IDENTITY_SYSTEM_LOWER="azure_ad"
 K8S_IMAGE_BASE="mcr.microsoft.com/k8s/azurestack/core/"
```

Comment:
remove this constant as we are now getting as input.

Reply:
Oops, I thought I took that out already